spec/support/playwright_global_teardown.js

```
@@ -0,0 +1,22 @@
+// Global teardown for Playwright tests in Rails 8 application
+
+async function globalTeardown() {
+  console.log('🎭 Cleaning up Playwright test environment...');
```

ESLint

[error] Unexpected console statement.

(no-console)